changelog.d/pr-8510

```diff
@@ -0,0 +1,8 @@
+synopsis: don't send non-extant extra-lib-dirs to ghc
```

Comment:
Didn't know that _non-existent_ can be condensed into _non-extant_ ... Now I know...